### PR TITLE
Add mrio_common_metadata

### PR DESCRIPTION
**Pull request by xylar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/mrio_common_metadata/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  home: https://github.com/brightway-lca/mrio_common_metadata
+  summary: Common Datapackage schema and utilities for MRIO tables
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file:
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - cmutel
```
Comment (xylar) [inline recipes/mrio_common_metadata/meta.yaml:47]: @cmutel, one more for you to confirm that you're willing to maintain.

Comment (cmutel) [reply recipes/mrio_common_metadata/meta.yaml:47]: Confirmed!

**Comment by xylar** [comment]
@conda-forge/help-python, this is also ready to review. Thank you in advance!

**Comment by xylar** [comment]
@chrisburr, you kindly reviewed other Brightway-related recipes recently. 
https://github.com/conda-forge/staged-recipes/pull/19941
https://github.com/conda-forge/staged-recipes/pull/19944
https://github.com/conda-forge/staged-recipes/pull/19946
 Would you have time to review this one as well?  It is a dependencies of other recipes that @cmutel would like to get in.

**Comment by xylar** [comment]
@conda-forge/help-python, we've been waiting on a review here for about 2 weeks now.  Any chance of getting one?

**Comment by xylar** [comment]
Thanks @chrisburr!

@cmutel, I think you wanted to do the next round of recipes yourself.  Just ping me if you want help or if you want to include me as a maintainer.

**Pull request merged**